.github/workflows/validate.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           go-version: stable
```

Comment:
We talked about using `oldstable` here to keep from having breakage with new Go releases. Thoughts on bumping it back to that?

Reply:
So the assumption is devs will discover breakages on their machines well before CI hits.  Ok.